### PR TITLE
Add SchemaRegistry to packages csv and release notes

### PR DESCRIPTION
**Pull request by MiYanni**
Updated packages csv and release notes for SchemaRegistry.

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -1,5 +1,6 @@
 "Package","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","Hide","Notes"
 "Azure.Data.AppConfiguration","1.0.1","","App Configuration","App Configuration","appconfiguration","","","client","",""
+"Azure.Data.SchemaRegistry","","1.0.0-beta.1","Schema Registry","Schema Registry","schemaregistry","","","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:3]: This change is going to conflict with our automation PR https://github.com/Azure/azure-sdk/pull/1776. You can rebase on top of that change and add in "client" as the Type if we want this to show up as a new library in the package index.

Comment (MiYanni) [reply _data/releases/latest/dotnet-packages.csv:3]: I only have Client for the actual client. For the other package (Avro one), it is not a client.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:3]: This entry did get added below by the automation. Can you delete that entry and keep yours?

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -115,6 +116,7 @@
 "Microsoft.Azure.Data.SchemaRegistry.ApacheAvro","","1.0.0-beta.1","Microsoft.Azure.Data.SchemaRegistry.ApacheAvro","","NA","NA","NA","","",""
 "Microsoft.Azure.DataLake.Store","","1.2.3-alpha","Microsoft.Azure.DataLake.Store","Data Lake","https://github.com/Azure/azure-data-lake-store-net/tree/1.2.3-alpha","https://docs.microsoft.com/dotnet/api/overview/azure/data-lake-store?view=azure-dotnet-preview","NA","","",""
 "Microsoft.Azure.DataLake.USQL.SDK","1.4.200831","","Microsoft.Azure.DataLake.USQL.SDK","","NA","NA","NA","","",""
+"Microsoft.Azure.Data.SchemaRegistry.ApacheAvro","","1.0.0-beta.1","Schema Registry - Apache Avro","Schema Registry - Apache Avro","schemaregistry","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:119]: Can you delete the other entry for this 3 links up?

Comment (MiYanni) [reply _data/releases/latest/dotnet-packages.csv]: Fixed.

**Comment by MiYanni** [comment]
@weshaggard Not sure how the links are being generated that cause the failure. It seems to create these, which go to nothing.
https://docs.microsoft.com/dotnet/api/overview/azure/Data.SchemaRegistry-readme-pre?view=azure-dotnet-preview
https://docs.microsoft.com/dotnet/api/overview/azure/Microsoft.Data.SchemaRegistry.ApacheAvro-readme-pre?view=azure-dotnet-preview

**Comment by weshaggard** [comment]
Those are the MSDocs links and the automation will usually put an "NA" in the msdocs column for any missing docs. Given this is a new library they might not have made it through the publishing pipeline yet. I suggest just marking them as "NA" for now and the automation will remove that once the docs get published.

**Comment by MiYanni** [comment]
@weshaggard The NA trick worked. Thank you!

**Comment by MiYanni** [comment]
@weshaggard Does this PR look good? I'll be out next week so I'd like to get it merged today if possible.

**Review by kinelski — APPROVED** [review]
LGTM.

**Comment by kinelski** [comment]
> 
> 
> Does this PR look good? I'll be out next week so I'd like to get it merged today if possible.

Looks fine, but I can address other issues if @weshaggard points something out.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**